pkg/apis/vpc/v1alpha1/subnet_types.go

```diff
@@ -8,15 +8,19 @@ import (
 )
 
 type AccessMode string
+type DHCPConfigMode string
 
 const (
-	AccessModePublic  string = "Public"
-	AccessModePrivate string = "Private"
-	AccessModeProject string = "PrivateTGW"
+	AccessModePublic          string = "Public"
+	AccessModePrivate         string = "Private"
+	AccessModeProject         string = "PrivateTGW"
+	DHCPConfigModeDeactivated string = "DHCP_DEACTIVATED"
```

Comment:
Please follow Go/K8s convention, and use "DHCPDeactivated", etc.
In NSX API, it is "DEACTIVATED" or "DHCP_DEACTIVATED"?

Reply:
Updated, thanks! It's DHCP_DEACTIVATED for now, while the patch is sill not merged. Ref: https://gitreview-vcfnet.lvn.broadcom.net/c/nsx/+/657239/14/mp/policy/policy-api/src/main/resources/api_spec/root/nsx-policy/policy-vpc-subnet.yml#316